oslabs-beta/Kubiquity
# Update metrics Controller

WHAT DOES THIS PR DO:
1. Converts the Prometheus work from the promise work to async/await. 
2. Minor data formatting to memory and CPU. 

HOW TO TEST THIS PR: 
1. Open Docker Desktop
2. Run `minikube start`
3. Run `gh pr checkout 47 && npm install && npm run dev`
4. Verify that the app still works, specifically the metrics. 
5. Verify that memory (bytes) is an integer. 
6. Verify that CPU use (%) has one decimal. 
7. For real verify 5 and 6 because I did not test it! ;) :D
